lms/djangoapps/edraak_marketing_email/models.py

```diff
@@ -0,0 +1,15 @@
+"""
+Edraak-Marketing-Email-related models.
+"""
+from django.db import models
+from django.contrib.auth.models import User
```

Comment:
Because it's swappable.

```suggestion
from django.contrib.auth import get_user_model
```